packages/bmodelcanvas/src/ui/vue.d.ts

```diff
@@ -0,0 +1,9 @@
+declare module "*.vue" {
+    import Vue from "vue";
+    export default Vue;
+}
+declare module "vuetify/lib/framework" {
+	import Vuetify from "vuetify";
+	import 'vuetify/dist/vuetify.min.css';
```

Comment:
Asta sigur trebuie importat aici? Eu cred ca ajunge daca il importi in main, insa nu in d.t.s.

Reply:
Partea de vue da, altfel nu recunoaste componentele .vue